server/sonar-web/src/main/js/components/controls/Checkbox.tsx

```diff
@@ -47,7 +47,8 @@ export default class Checkbox extends React.PureComponent<Props> {
     const className = classNames('icon-checkbox', {
       'icon-checkbox-checked': this.props.checked,
       'icon-checkbox-single': this.props.thirdState,
-      'icon-checkbox-disabled': this.props.disabled
+      'icon-checkbox-disabled': this.props.disabled,
+      'text-muted': this.props.disabled
```

Comment:
This class should be applied to the `<a />` element and not to checkbox.

Please make sure that the link will actually be greyed out.